commands/management/add_mysql.go

```diff
@@ -176,6 +180,35 @@ func (cmd *addMySQLCommand) Run() (commands.Result, error) {
 		}
 	}
 
+	if cmd.DefaultsFile != "" {
+		path := cmd.DefaultsFile
+
+		type Config struct {
+			Client struct {
+				User     string
+				Password string
+			}
+		}
+
+		if strings.HasPrefix(path, "~/") {
```

Comment:
Is it necessary step?

Reply:
When I ran:
./bin/pmm-admin add mysql --defaults-file=~/my.sandbox.cnf MyDB 127.0.0.1:3306
without that, then I got:
open ~/my.sandbox.cnf: no such file or directory

Also I google little bit and found more people which do this. So I think yes, but I am Windows user, so I can be wrong.